package.json

```diff
@@ -21,13 +21,12 @@
   },
   "homepage": "https://github.com/driftyco/ionic-module-template#readme",
   "devDependencies": {
```

Comment:
So I mostly have it working. Had some AoT problems as usual, but got it compiling in the end. However, I can only get inline templates to work, not sure why. Also CSS is missing completely.

The documentation on AoT is as sparse as ever.